src/content/docs/apm/agents/java-agent/getting-started/monitor-your-java-app.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Monitor your Java application
+metaDescription: "Want to start monitoring your Java application? Find the best way to get started here."
+redirects:
+  - /docs/agents/java-agent/installation/install-java-agent
+  - /docs/java/wildfly-installation-for-java
+  - /docs/docs/installing_the_java_agent_on_websphere
+  - /docs/java/installing-the-java-agent-on-websphere
+  - /docs/java/websphere-installation-for-java
+  - /docs/java/installing-the-java-agent-on-weblogic
+  - /docs/java/weblogic-installation-for-java
+  - /docs/docs/tomcat_windows_agent
+  - /docs/java/installing-the-java-agent-on-tomcat-and-windows
+  - /docs/java/the-apache-commons-daemon-jsvc
+  - /docs/java/tomcat-installation-for-java
+  - /docs/java/play-installation-for-java
+  - /docs/java/installing-the-java-agent-on-jboss
+  - /docs/java/jboss-installation-for-java
+  - /docs/java/self-installing-java-agent
+  - /docs/general/self-installing-java-agent
+  - /docs/java/java-agent-self-installer
+  - /docs/java/java-agent-manual-installation
+  - /docs/java/grails-installation-for-java
+  - /docs/docs/installing_java_agent_on_glassfish
+  - /docs/java/installing-the-java-agent-on-glassfish
+  - /docs/java/glassfish-installation-for-java
+  - /docs/java/installing-the-java-agent-on-coldfusion
+  - /docs/java/coldfusion-installation-for-java
+  - /docs/agents/java-agent/frameworks/glassfish-installation-java
+  - /docs/agents/java-agent/frameworks/jboss-installation-java
+  - /docs/agents/java-agent/frameworks/coldfusion-installation-java
+  - /docs/agents/java-agent/frameworks/play-installation-java
+  - /docs/agents/java-agent/frameworks/tomcat-installation-java
+  - /docs/agents/java-agent/frameworks/weblogic-installation-java
+  - /docs/agents/java-agent/frameworks/websphere-installation-java
+  - /docs/agents/java-agent/frameworks/websphere-liberty-profile-installation-java
+  - /docs/agents/java-agent/frameworks/wildfly-installation-java
+  - /node/2596
+  - /docs/agents/java-agent/frameworks/grails-installation-java
+  - /node/2306
+  - /docs/agents/java-agent/installation/java-agent-manual-installation
+  - /docs/agents/java-agent/installation/java-agent-self-installer
+  - /docs/apm/agents/java-agent/installation/install-java-agent
+---
+
+import javaApmExampleImage from 'images/java-apm-example-dashboards.png'
+
+Want to start monitoring your Java application with New Relic right away? Use this guide to find the best path to start sending your application data to New Relic.
```

Comment:
@paperclypse, here's a comment I'm testing right in the DIFF view of CodeStream!

Reply:
I've finally had time to get to these! Thanks for all your comments, @rhetoric101!